client/src/components/Map.jsx

```diff
@@ -0,0 +1,226 @@
+import { MapContainer, TileLayer, Polygon, Marker } from "react-leaflet";
+import { useState, useEffect } from "react";
+import { useHistory } from "react-router-dom";
+function Map(props) {
+  const [center, setCenter] = useState([37.55791, -77.453143]);
+  const [clickMarker, setClickMarker] = useState();
+  const [clickLocation, setClickLocation] = useState();
+  //array of json objects
+  const [foodList, setFoodList] = useState(null);
+  clickMarker
+    ? console.log(` Click Marker outside fetch = ${clickMarker.id}`)
+    : console.log(`Marker has not been clicked yet`);
+  let interLocation = [];
+
+  // const fetchRestData = async () => {
+  //     const response = await fetch('api/')
+  // }
+
+  //provides access to current url and allows me to push a new route
+  let history = useHistory();
+
+  //state of
+
+  //   useEffect(() => {
+  //     if (!clickMarker) {
+  //       fetch("/api/restaurants")
+  //         .then((res) => res.json())
+  //         .then((resList) => {
+  //           setFoodList(resList);
+  //         });
+  //     }
+  //   });
+
+  //   console.log(props.restaurantId);
+  //   console.log("click = " + clickMarker.name);
+  return (
+    <>
+      <MapContainer
+        center={center}
+        zoom={13}
+        scrollWheelZoom={true}
+        zoomControl={true}
+        touchZoom={true}
+        dragging={true}
+        style={{ height: "70vh", width: "60vw" }}
+      >
+        <TileLayer
+          url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+          attribution='<a href="https://github.com/cyclosm/cyclosm-cartocss-style/releases" title="CyclOSM - Open Bicycle render">CyclOSM</a> | Map data: &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+        />
+        //Kuba Kuba
```

Comment:
Inline comments in React components should be wrapped in curly brackets, forward slash and an asterix like this: `{/*whatever the comment*/}`, you can also wrap what ever trying to comments by highlighting it and use the command `ctrl` + `/`